client/src/app/pages/migration-waves/components/migration-wave-form.tsx

```diff
@@ -181,7 +173,8 @@ export const WaveForm: React.FC<WaveFormProps> = ({
   } = useForm<WaveFormValues>({
     mode: "onChange",
     defaultValues: {
-      name: migrationWave?.name || "",
+      name:
+        migrationWave && migrationWave.name !== null ? migrationWave.name : "",
```

Comment:
I'm pretty sure this change has no effect, since a `null` name value is falsy and would cause `migrationWave?.name || ""` to evaluate to `""`. I would vote to revert this for simplicity.

Reply:
Right, I restored it.